README.md

```diff
@@ -2160,14 +2160,39 @@ To verify a deployed [`CreateX`](./src/CreateX.sol) contract on a block explorer
 ## [`CreateX`](./src/CreateX.sol) Deployments
 
 > [!TIP]
-> The complete list with additional chain information per deployment can be retrieved via [createx.rocks](https://createx.rocks).
+> The complete list with additional chain information per deployment can be retrieved via [createx.rocks](https://createx.rocks). In addition, all [sourcify.eth](https://sourcify.dev) verification links can be found in the [`deployments.json`](./deployments/deployments.json) file.
 
 - EVM-Based Production Networks:
-  - Ethereum: [`0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed`](https://etherscan.io/address/0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed)
+  - Ethereum: TBD
 - Ethereum Test Networks:
   - Sepolia: [`0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed`](https://sepolia.etherscan.io/address/0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed)
+  - Holešky (Holešovice): [`0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed`](https://holesky.etherscan.io/address/0xba5ed099633d3b313e4d5f7bdc1305d3c28ba5ed)
```

Comment:
What do you think about leaving out the address for each entry unless it's different (e.g. on zkSync or other incompatible chains), to reduce noise?
```suggestion
  - [Holešky (Holešovice)]((https://holesky.etherscan.io/address/0xba5ed099633d3b313e4d5f7bdc1305d3c28ba5ed))
```

Reply:
Yeah, that's fine I think. I pinned the address in the section again, so people can simply copy it. Resolved via https://github.com/pcaversaccio/createx/commit/d5f26737ddbac3ffe9e32410d96084fcd42ea126.